app/services/custom_landing_page/markdown_helper.rb

```diff
@@ -0,0 +1,87 @@
+module CustomLandingPage
+
+  # Limited HTML Markdown Renderer extend the default
+  # Redcarpet HTML rendered, but it ignores certain Markdown block.
+  #
+  # See the list of IGNORED_BLOCKS to see which blocks are ignored.
+  # The commented blocks in the list are available.
+  #
+  class LimitedHTMLMarkdownRenderer < Redcarpet::Render::HTML
+
+    IGNORED_BLOCKS = [
+      'block_code',
+      'block_quote',
+      'block_html',
+      'footnotes',
+      'footnote_def',
+      'header',
+      'hrule',
+      'list',
+      'list_item',
+      # 'paragraph',
+      'table',
+      'table_row',
+      'table_cell',
+      'autolink',
+      'codespan',
+      # 'double_emphasis',
+      # 'emphasis',
+      'image',
+      # 'linebreak',
+      # 'link',
+      'raw_html',
+      # 'triple_emphasis',
+      # 'strikethrough',
+      'superscript',
+      # 'underline',
+      'highlight',
+      'quote',
+      'footnote_ref',
+    ].each do |ignored_block|
+      define_method ignored_block do |*|
+        nil
+      end
+    end
+  end
+
+  module MarkdownHelper
+
+    # Main method to render Markdown
+    def render_markdown(str)
+      markdown.render(str).html_safe
+    end
+
+    # Memoize the Markdown instance, as instructed in
+    # the Redcarpet documentation.
+    #
+    # Read more: https://github.com/vmg/redcarpet#and-its-like-really-simple-to-use
+    #
+    def markdown
+      @markdown ||= Redcarpet::Markdown.new(
+        LimitedHTMLMarkdownRenderer,
+        strikethrough: true,
+        underline: true,
+        filter_html: true, # do not allow any user-inputted HTML in the output.
+        no_images: true, # do not generate any <img> tags.
+        no_styles: true, # do not generate any <style> tags.
+        safe_links_only: true # only generate links for protocols which are considered safe.
+      )
+    end
+
+    # Deprecated
```

Comment:
I didn't spot a change that removes a mention of array from docs but since it's deprecated and if it is mentioned in any documentation or example code, it should be removed from there.